integrations/express/middleware.ts

```diff
@@ -94,6 +95,19 @@ export default function middleware(
       return;
     }
 
+    //   the user is makiing the request so we made the fun - off , i.e the keploy server is in off mode 
+    const fun = keploy.mode.GetMode();
```

Comment:
You are correct. we should also set mode in ctx to off. 
 Can you do the logic expression for if stmt in a single line of code here.